crossplane/function/runtime.py

```diff
@@ -17,6 +17,8 @@
 import asyncio
 import os
 import signal
+from collections.abc import Sequence
+from typing import Any
```

Comment:
We typically try to avoid importing types directly, but instead only import packages. I thought our linter should catch this. I'll update it.